tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs

```diff
@@ -33,7 +32,7 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
             }
         }
 
-        [Theory(Skip = "Bug#1259616")]
+        [Theory]
```

Comment:
How did we resolve this test? Is the reason it was failed because we didn't use `ORYXTESTS_LTS_VERSIONS_BUSTER_BUILDIMAGE_DOCKERFILE` test image in this test?

Reply:
no .. i had to trigger publish binaries to prod ... without that those versions were only in dev storage account